src/veh_type.h

```diff
@@ -127,6 +132,33 @@ class vpart_info
         /** Mechanics skill required to install item */
         int difficulty = 0;
 
+        /** Legacy parts don't specify installation requirements */
+        bool legacy = true;
+
+        /** Installation requirements for this component */
+        requirement_data install_reqs;
+
+        /** Required skills to install this component */
+        std::map<skill_id, int> install_skills;
+
+        /** Installation time (in moves) for component (@see install_time), default 1 hour */
+        int install_moves = 60000;
```

Comment:
The task (convert time to moves) isn't rare in the code. I suppose it deserves yet another macro (`constexpr` function) `MOVES( HOURS( 1 ) )`.